ansible_collections/arista/avd/roles/eos_validate_state/templates/generate_vars_for_testing.j2

```diff
@@ -8,4 +8,16 @@
 {%     endif %}
 {% endfor %}
 loopback0_reachability:
-  loopback0_range: {{ address | unique }}
\ No newline at end of file
+  loopback0_range: {{ address | unique }}
```

Comment:
This file should be removed at the end of the role execution as it is not a file to expose to user.

Reply:
Done. This file is now removed at the end of the tests.